saskatoon/sitebase/templates/registration/base.html

```diff
@@ -0,0 +1,83 @@
+{% load i18n %}
```

Comment:
Just curious, do we really need to create a new base? why not extending from app/base/view.html?
Are stylesheets very different from the ones needed in the app?

Reply:
i just moved existing tags in `login.html`, didn't realise most of the tags were the same as `base/view.html`.

refactored `base` to make a template with an empty body. let me know if that works

Reply:
ok sweet, how about we call your barebone base `app/base/base.html` instead of `view_empty` ?

Reply:
done!